### DISCUSSION TITLE
Shortkey for enable / disable survey

### DISCUSSION THREAD

**Discussion by tammojan** [Stellarium/stellarium#213] [Q&A] [answered]
Is it possible to toggle showing the latest used survey with some shortkey? I.e. when I press H, a hydrogen sky appears, and when I press H again, I'm back to 'normal' stellarium without any survey.

**Comment by gzotti** [comment, marked as answer]
The Guide answers this on page 16, and you can assign your own hotkeys (section 4.7.1).

**Comment by alex-w** [comment]
Default shortcut for toggle HiPS is Ctrl+Alt+D, but you may change it in Shortcut Editor (Details in Stellarium User Guide).

**Comment by tammojan** [comment]
Thanks for your replies. Page 16 indeed mentions the HiPS-button; I just found out that I have to enable it through Control Panel / Extras. Pressing the button does exactly what I want; unfortunately the shortkey does not work. Also manually setting a different shortcut key, pressing the key does not toggle HiPS.

**Comment by gzotti** [comment]
Have you enabled a survey in the first place? 
The default hotkey works on my Windows systems with German keyboard layout. Your mileage may vary.

**Comment by tammojan** [comment]
Apparently my mileage varies, the shortkey does not work. Using it with Stellarium 0.18.1, macos, US International keyboard layout. Probably because Cmd-Option-D gets caught by the system to toggle the Dock. Changing it to Cmd-Option-P does work.
